### PR TITLE
Friday at grange

### PR DESCRIPTION
**Pull request by dbwest**
Some stuff in there, like activate and deactivate returning self came from me messing around.... We don't need to keep that. I just wanted to get you the Friday stuff ASAP

### CODE REVIEW

File: CORE/elements/text_field.rb [hunk 1]
```diff
@@ -18,25 +16,11 @@ def value
...
-        end
-        @world.ledger.record_fill(@name, data)
-    end
-
     def manually_clear
       watir_element.click
       browser.send_keys(:end)
       watir_element.value.size.times do
-          browser.send_keys(:backspace) unless watir_element.value == ""
+          browser.send_keys(:backspace) unless watir_element.value.chomp == ""
```
Comment (greenarrowdb) [inline CORE/elements/text_field.rb]: Why did we add this chomp here? I don't think this should be a thing. If the element has empty whitespace, we still want to clear that whitespace out, not ignore it.

File: CORE/elements/text_field.rb [hunk 2]
```diff
@@ -1,6 +1,4 @@
-
-
-class TextFieldElement < CoreElement
+class TextFieldElement < FillableElement
```
Comment (greenarrowdb) [inline CORE/elements/text_field.rb]: Remove the fillableElement classing please. this is not saving us any work right now, it is just adding another class to keep track of.

File: .gitignore [hunk 3]
```diff
@@ -1,7 +1,10 @@
-
```
Comment (greenarrowdb) [inline .gitignore]: I added another comment banner here for clarity but these are good additions :)

File: CORE/elements/text_field.rb (hunk 1 above)
Comment (dbwest) [reply CORE/elements/text_field.rb]: that should not be there. will take it out. we were debugging and we put it in for some reason

**Comment by dbwest** [comment]
I removed fillable and chomp

**Comment by greenarrowdb** [comment]
Looks like Fillable and Chomp are still there, did you push your changes?

**Comment by dbwest** [comment]
oh no
let me check on that

**Comment by dbwest** [comment]
It had rejected my push. Had to do another pull before I pushed again...

**Pull request merged**